frontend/src/metabase/visualizations/components/ClickActions/ClickActionControl.tsx

```diff
@@ -113,5 +114,9 @@ export const ClickActionControl = ({
       return <InfoControl>{action.title}</InfoControl>;
   }
 
+  if (isCustomClickActionWithView(action)) {
```

Comment:
My VS code reports this part is unreachable
![image](https://github.com/metabase/metabase/assets/1937582/20b2567f-10ae-4ff3-903b-1fea2d4f4677)

Because all `buttonType` has been covered in the `switch` statement. If this is still needed then I think there would be some typing problems to fix first.